plugin-client/src/main/java/com/cloudera/nav/plugin/client/NavApiCient.java

```diff
@@ -71,18 +78,105 @@ public void registerModels(Collection<Object> models) {
    * @return a collection of available sources
    */
   public Collection<Source> getAllSources() {
+    System.out.println("here");
     RestTemplate restTemplate = new RestTemplate();
-    String url = getSourceUrl();
+    String url = getUrl();
     HttpHeaders headers = getAuthHeaders();
+    System.out.println(headers);
     HttpEntity<String> request = new HttpEntity<String>(headers);
+    Class<SourceAttrs[]> sourceAttrsClass = SourceAttrs[].class;
     ResponseEntity<SourceAttrs[]> response = restTemplate.exchange(url,
-        HttpMethod.GET, request, SourceAttrs[].class);
+        HttpMethod.GET, request, sourceAttrsClass);
+//    ResponseEntity<SourceAttrs[]> response = restTemplate.getForEntity(url, )
+    System.out.println(response);
```

Comment:
same here, don't use System.out.println. If you want to leave this in, convert it to LOG.debug